src/main/twirl/views/manage/guidance.scala.html

```diff
@@ -3,7 +3,7 @@
 )
 
 
-@main(s"Guidance on grants - RIFS", displayUserName=Some("Portfolio Pete"),
+@main(s"Guidance on grants - RIFS", displayUserName=Some("Portfolio Manager"),
```

Comment:
I thought we were using the persona's for this to keep it in keeping with the applicant journey "Experienced Eric"?  I've spoken to Gerald he agrees it should be "Portfolio Peter".